mappings/net/minecraft/server/world/ChunkStorage.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_5318 net/minecraft/server/world/ChunkStorage
+	METHOD method_27476 loadChunk (Lnet/minecraft/class_5086;II)Lnet/minecraft/class_3607;
+		ARG 1 world
+	METHOD method_27477 saveChunk (Lnet/minecraft/class_5086;Lnet/minecraft/class_3607;)V
```

Comment:
Just `save` ?

Reply:
I think `saveChunk` is clearer since there's also `saveEntities`

Reply:
hmm lets go with that then.